### ISSUE TITLE
SmartCity-Accessibility #168 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#117]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 33 lines...]
Downloading: https://jitpack.io/teamdev/jxmaps-linux64/1.2/jxmaps-linux64-1.2.pom

Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/teamdev/jxmaps-linux64/1.2/jxmaps-linux64-1.2.pom

[WARNING] The POM for teamdev:jxmaps-linux64:jar:1.2 is missing, no dependency information available
Downloading: https://jitpack.io/teamdev/jxmaps-mac/1.2/jxmaps-mac-1.2.pom

Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/teamdev/jxmaps-mac/1.2/jxmaps-mac-1.2.pom

[WARNING] The POM for teamdev:jxmaps-mac:jar:1.2 is missing, no dependency information available
Downloading: https://jitpack.io/teamdev/jxmaps-win/1.2/jxmaps-win-1.2.pom

Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/teamdev/jxmaps-win/1.2/jxmaps-win-1.2.pom

[WARNING] The POM for teamdev:jxmaps-win:jar:1.2 is missing, no dependency information available
Downloading: https://jitpack.io/teamdev/license/1.0/license-1.0.pom

Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/teamdev/license/1.0/license-1.0.pom

[WARNING] The POM for teamdev:license:jar:1.0 is missing, no dependency information available
Downloading: https://jitpack.io/com/nimbusds/lang-tag/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/lang-tag/maven-metadata.xml


Downloading: https://jitpack.io/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml
Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/com/nimbusds/nimbus-jose-jwt/maven-metadata.xml


Downloading: https://jitpack.io/teamdev/jxmaps/1.2.2/jxmaps-1.2.2.jar

Downloading: file:///var/lib/jenkins/workspace/SmartCity-Accessibility/libs/teamdev/jxmaps/1.2.2/jxmaps-1.2.2.jar

Downloading: https://repo.maven.apache.org/maven2/teamdev/jxmaps/1.2.2/jxmaps-1.2.2.jar

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 8.591 s
[INFO] Finished at: 2016-12-29T18:54:29+00:00
[INFO] Final Memory: 14M/106M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project SmartCity-Accessibility: Could not resolve dependencies for project SmartCity-Accessibility:SmartCity-Accessibility:jar:0.0.1-SNAPSHOT: Could not find artifact teamdev:jxmaps:jar:1.2.2 in jitpack.io (https://jitpack.io) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 2ad1422a94e088ff8b55bb0ec1717f93d3620d1f - updated pom to latest jxmaps #114

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/168/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
